### PR TITLE
Google cloud build

### PR DESCRIPTION
**Pull request by andrewkandzuba**
- Continues Integration for Google Cloud Build

### CODE REVIEW

File: deployment.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ spec:
         env: canary
     spec:
       containers:
-        - image: REPLACE_IMAGE
+        - image: DOCKER_IMAGE
```
Comment (balamuru) [inline deployment.yaml:18]: what does DOCKER_IMAGE refer to ? Is it a placeholder ?

File: namespace.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: GKE_NAMESPACE
\ No newline at end of file
+  name: KUBECTL_NAMESPACE
```
Comment (balamuru) [inline namespace.yaml:4]: Is this a placeholder ? Or do you intend the user to replace this? It's not a helm chart. Not clear.

**Review by balamuru — CHANGES_REQUESTED** [review]
Please clarify the comments / questions

File: namespace.yaml (hunk 2 above)
Comment (andrewkandzuba) [reply namespace.yaml:4]: It is placeholder replaced in bash script https://github.com/andrewkandzuba/openexchange-webservice-go/blob/google-cloud-build/deploy.sh#L4

File: deployment.yaml (hunk 1 above)
Comment (andrewkandzuba) [reply deployment.yaml:18]: It is a placeholder replaced in bash script https://github.com/andrewkandzuba/openexchange-webservice-go/blob/ddf4bfd1ff704cb8d520eedbdea97b12be721867/deploy.sh#L6

File: README.md [hunk 3]
```diff
@@ -84,13 +84,41 @@ Following environment variables are mandatory for any successful build:
     - We use Google Cloud SDK service account authentication method [with key file](https://cloud.google.com/sdk/gcloud/reference/auth/activate-service-account) method to get access to the GKE cluster. 
     - The generated key file should be included into the project but in encrypted form. Travis CI supports [file encryption](https://docs.travis-ci.com/user/encrypting-files/) that greatly facilitates and simplify the solution.
     - Please refer to this example `travis encrypt-file super_secret.txt --add` to understand how the encryption works and how Travis populates these two environment variable.
-- **GKE_PROJECT** - the name of target Google Cloud Project.
-- **GKE_ZONE** - the name of the target GCP cluster zone.
-- **GKE_CLUSTER** - the name of the target GCP cluster. Note that the cluster must be precreated. See "_1.2. Deploy to Google Kubernetes Engine (GKE)_"
-- **GKE_NAMESPACE** - the target Kubernetes namespace. 
+- **CLOUDSDK_CORE_PROJECT** - the name of target Google Cloud Project.
```
Comment (balamuru) [inline README.md:89]: don't you need a doc entry for DOCKER_IMAGE too?

File: deployment.yaml (hunk 1 above)
Comment (balamuru) [reply deployment.yaml:18]: see new comment in README.md (i.e. document this better)

File: namespace.yaml (hunk 2 above)
Comment (balamuru) [reply namespace.yaml:4]: ok

File: README.md (hunk 3 above)
Comment (andrewkandzuba) [reply README.md:89]: I want to calculate this using [Github API](https://developer.github.com/v3/)

File: deployment.yaml (hunk 1 above)
Comment (andrewkandzuba) [reply deployment.yaml:18]: Added note about `DOCKER_IMAGE` environment variable into [this section](https://github.com/andrewkandzuba/openexchange-webservice-go/tree/google-cloud-build#12-deploy-to-google-kubernetes-engine-gke)

File: README.md (hunk 3 above)
Comment (andrewkandzuba) [reply README.md:89]: Added note about `DOCKER_IMAGE` environment variable into [this section](https://github.com/andrewkandzuba/openexchange-webservice-go/tree/google-cloud-build#12-deploy-to-google-kubernetes-engine-gke)

**Comment by andrewkandzuba** [comment]
https://coveralls.io is down so cannot build it right now ...

**Comment by andrewkandzuba** [comment]
@balamuru - Are you still going to add comments to that? I am about to merge it `master` branch.

**Pull request merged**